docs/data-sources/equinix_fabric_cloud_router.md

```diff
@@ -23,29 +23,31 @@ data "equinix_fabric_cloud_router" "cloud_router_data_name" {
 <!-- schema generated by tfplugindocs -->
```

Comment:
Look into the issues for this repo and reference any of them that are related to Cloud Router. I believe there's one: https://github.com/equinix/terraform-provider-equinix/issues/423

But double check for others. We can just put that in the main conversation of the PR.